src/Controller/EnMarche/MunicipalChiefController.php

```diff
@@ -14,21 +19,105 @@
  * @Route("/espace-chef-municipal")
  * @Security("is_granted('ROLE_MUNICIPAL_CHIEF')")
  */
-class MunicipalChiefController extends AbstractController
+class MunicipalChiefController extends Controller
 {
     use CanaryControllerTrait;
 
     /**
-     * @Route("/", name="app_municipalchief_index")
+     * @Route(
+     *     path="/municipale/candidature-colistiers",
+     *     name="app_municipalchief_municipal_runningmaterequest",
+     *     methods={"GET"},
+     * )
      */
-    public function indexAction(UserInterface $municipalChief): Response
-    {
+    public function municipalRunningMateRequestAction(
+        RunningMateRequestRepository $runningMateRequestRepository,
+        MunicipalExporter $municipalExporter,
+        UserInterface $municipalChief
+    ): Response {
         $this->disableInProduction();
 
-        return $this->render('municipal_chief/index.html.twig', [
+        return $this->render('municipal_chief/municipal/running_mate/list.html.twig', [
             'managedCities' => FranceCitiesBundle::searchCitiesByInseeCodes(
                 $municipalChief->getMunicipalChiefManagedArea()->getCodes()
             ),
+            'runningMateListJson' => $municipalExporter->exportRunningMateAsJson(
+                $runningMateRequestRepository->findForMunicipalChief($municipalChief),
+                'app_municipalchief_municipal_runningmaterequest_detail'
+            ),
+        ]);
+    }
+
+    /**
+     * @Route(
+     *     path="/municipale/candidature-benevole",
+     *     name="app_municipalchief_municipal_volunteerrequest",
```

Comment:
why not `name="app_municipal_chief_municipal_volunteer_request",`?

Reply:
I found it shorter and more like app prefix + lowercased controller name + route path. I thought this was nice.

Reply:
agree with @nicoweb, snake case rule says `_` between 2 words